src/authorization/authorization.ts

```diff
@@ -425,15 +429,58 @@ export function initialize(
     }
   };
 
+  const getMergeDefaultValue = (merge?: boolean) => {
+    const doesAnonUserExist = getAnonUserId() === null;
+    return merge === undefined
```

Comment:
it's easier to read non-nested ternaries. Can we update this to use basic `if` statements instead?